doc/guides/west/without-west.rst

```diff
@@ -3,30 +3,58 @@
 Using Zephyr without west
 #########################
 
-There are many valid usecases to avoid using a meta-tool such as west which has
```

Comment:
I actually liked the description of the usecases here, can we keep it?

Reply:
Sure!